docs/bigtable.txt

```diff
@@ -0,0 +1,72 @@
+[[bigtable]]
+Google Cloud Bigtable
```

Comment:
Please use header styles with the number of `=` signs equal to the header level:
http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles

The method you're using (which is used throughout the docs) I'm fixing in a separate PR, as it's very confusing and brittle, as described in http://asciidoctor.org/docs/asciidoc-recommended-practices/#section-titles